docs/pt/Building-Alluxio-Master-Branch.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: global
+title: Construindo o Alluxio Master Branch
+nickname: Construindo o Master Branch
+group: Resources
+---
+
+* Table of Contents
+{:toc}
+
+Este guia descreve os passos para compilar o Alluxio do início.
+
+O pré-requisito para este guia é que você tenha [Java 7 (or above)](Java-Setup.html),
```

Comment:
Or I can go ahead and translate these others links that are missing. So we place this PR until I go them finished?